source/state_representation/include/state_representation/robot/JointState.hpp

```diff
@@ -268,27 +264,28 @@ class JointState : public State {
   void set_zero();
 
   /**
-   * @brief Clamp inplace the magnitude of the a specific state variable (velocities, accelerations or forces)
-   * @param max_absolute_value the maximum absolute magnitude of the state variable
+   * @brief Clamp inplace the magnitude of the a specific joint state variable
+   * @param max_absolute_value the maximum absolute value of the state variable
    * @param state_variable_type name of the variable from the JointStateVariable structure to clamp
-   * @param noise_ratio if provided, this value will be used to apply a dead zone under which
-   * the velocity will be set to 0
+   * @param noise_ratio if provided, this value will be used to apply a dead zone relative to the maximum absolute value
+   * under which the state variable will be set to 0
    */
-  void clamp_state_variable(double max_absolute_value,
-                            const JointStateVariable& state_variable_type,
-                            double noise_ratio = 0);
+  void clamp_state_variable(
+      double max_absolute_value, const JointStateVariable& state_variable_type, double noise_ratio = 0
+  );
 
   /**
-   * @brief Clamp inplace the magnitude of the a specific state variable (velocities, accelerations or forces)
-   * for each individual joints
-   * @param max_absolute_value_array the maximum absolute magnitude of the state variable for each joints individually
+   * @brief Clamp inplace the magnitude of the a specific joint state variable
+   * for each individual joint
+   * @param max_absolute_value_array the maximum absolute value of the state variable for each joints individually
    * @param state_variable_type name of the variable from the JointStateVariable structure to clamp
-   * @param noise_ratio_array if provided, this value will be used to apply a dead zone under which
-   * the velocity will be set to 0
+   * @param noise_ratio_array those values will be used to apply a dead zone relative to the maximum absolute value
+   * under which the state variable will be set to 0 for each individual joint
    */
-  void clamp_state_variable(const Eigen::ArrayXd& max_absolute_value_array,
-                            const JointStateVariable& state_variable_type,
-                            const Eigen::ArrayXd& noise_ratio_array);
+  void clamp_state_variable(
+      const Eigen::ArrayXd& max_absolute_value_array, const JointStateVariable& state_variable_type,
```

Comment:
should the second argument be lined break as well? I guess all of those are the result of the new formatting rules